src/chainstatemanager_opts.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022 The Bitcoin Core developers
```

Comment:
which folder should this file live in and why?

Reply:
I don't really much mind where this file lives, so I'm happy to move the file wherever reviewers deem appropriate!

Reply:
I would have assumed it will be located in kernel, based on your comment

> In the longer run, I think eventually (very far off when libbitcoincarl is very minimal) we could put everything that links into libbitcoincarl into the src/kernel/ folder.

https://github.com/bitcoin/bitcoin/pull/25065#discussion_r865910970

Reply:
Yeah true it would eventually be in `kernel/`, so I can just move it there now!

> libbitcoincarl

💀

Reply:
Addressed in 43f343811cfb22739f5785a00f5a09d7b599fe04